index.html

```diff
@@ -1,14 +1,459 @@
+              </p>
+            </li>
+            <li
+              class="p-3 col-md-2 row m-1 justify-content-center align-items-center"
+            >
+              <img
+                src="./visuals/program_icon_02.png"
+                alt="podium"
+                class="pt-md-2 col-md-auto col-2 program-icon"
+              />
+              <h6
+                class="col-md-12 text-start text-md-center col-3 orange-font fontS-prog-14 mb-0 m-md-2"
+              >
+                GDG content
+              </h6>
+              <p
+                class="col-md-12 col-7 ps-1 ps-md-0 text-start text-md-center fontS-127"
+              >
+                Google technologies in areas like Android, Flutter and so much
+                more!
+              </p>
+            </li>
+            <li
+              class="p-3 col-md-2 row m-1 justify-content-center align-items-center"
+            >
+              <img
+                src="./visuals/program_icon_03.png"
+                alt="podium"
+                class="pt-md-2 col-md-auto col-2 program-icon"
+              />
+              <h6
+                class="col-md-12 text-start text-md-center col-3 orange-font fontS-prog-14 mb-0 m-md-2"
+              >
+                GDG Forum
+              </h6>
+              <p
+                class="col-md-12 col-7 ps-1 ps-md-0 text-start text-md-center fontS-127"
+              >
+                Share your thoughts and opinions with experts for each topic.
+              </p>
+            </li>
+            <li
+              class="p-3 col-md-2 row m-1 justify-content-center align-items-center"
+            >
+              <img
+                src="./visuals/program_icon_04.png"
+                alt="podium"
+                class="pt-md-2 col-md-auto col-2 program-icon"
+              />
+              <h6
+                class="col-md-12 text-start text-md-center col-3 orange-font fontS-prog-14 mb-0 m-md-2"
+              >
+                Workshops
+              </h6>
+              <p
+                class="col-md-12 col-7 ps-1 ps-md-0 text-start text-md-center fontS-127"
+              >
+                Full-day workshops, hands-on experiences, and debugging events.
+              </p>
+            </li>
+            <li
+              class="p-3 col-md-2 row m-1 justify-content-center align-items-center"
+            >
+              <img
+                src="./visuals/program_icon_05.png"
+                alt="podium"
+                class="pt-md-2 col-md-auto col-2 program-icon"
+              />
+              <h6
+                class="col-md-12 text-start text-md-center col-3 orange-font fontS-prog-14 mb-0 m-md-2"
+              >
+                Networking
+              </h6>
+              <p
+                class="col-md-12 col-7 ps-1 ps-md-0 text-start text-md-center fontS-127"
+              >
+                Get opportunities to network with Coders around the world.
+              </p>

```

Comment:
- [ ] Kindly implement the hover effect for each card as shown in the design template given.

![image](https://user-images.githubusercontent.com/77106962/225112361-266839b4-b704-48db-8b54-3017ccf0085f.png)